test/api_test.js

```diff
@@ -100,4 +116,9 @@ suite('api', () => {
     assert.equal(builds.builds[0].repository, 'xyz');
     assert.equal(builds.builds[0].sha, 'y650871208002a13ba35cf232c0e30d2c3d64783');
   });
+
+  test('integration installation', async function() {
+    let result = await helper.github.isRepoSet('qwerty', 'coolRepo');
+    assert.deepEqual(result, {installed: true});
+  });
```

Comment:
This suite should probably test the other two cases, too:
 * no such org known
 * org is known but the repo is not installed